test/unit/test.js

```diff
@@ -5,7 +5,7 @@ import Vuex, { mapState, mapMutations, mapGetters, mapActions } from '../../dist
 Vue.use(Vuex)
 
 const TEST = 'TEST'
-const TEST2 = 'TEST2'
+// const TEST2 = 'TEST2'
```

Comment:
eslint complained that this constant was never used anywhere in the tests so i commented it out. Should I remove it, or is it needed for something? Tests all worked without it.